db/migrate/20230608154821_add_unique_index_to_invoice_subscriptions.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddUniqueIndexToInvoiceSubscriptions < ActiveRecord::Migration[7.0]
+  def change
+    add_index :invoice_subscriptions,
+      %i[subscription_id from_datetime to_datetime recurring],
```

Comment:
Should'nt `recurring` be in the the `where` condition rather than in the index itself?